src/components/VAudioTrack/VWaveform.vue

```diff
@@ -258,7 +258,7 @@ export default defineComponent({
      */
     const timeFmt = (seconds: number): string => {
       const date = new Date(0)
-      date.setSeconds(seconds)
+      date.setSeconds(isNaN(seconds) ? 0 : seconds)
```

Comment:
Given that the way `isNaN` works can be confusing, (even the `Number.isNaN` isn't straightforward usecase for this) it might make more sense to use `Number.isFinite` and reverse the order:

```suggestion
      date.setSeconds(Number.isFinite(seconds) ? seconds : 0)
```

@sirreal and I were chatting offline the other day about potentially trying to find eslint rules for the projects we work on to prevent using the legacy `window.isNaN` and `window.isFinite` functions, neither of them works entirely as expected :cold_sweat: